libs/core/src/utils/test/truncation-tooltip.spec.ts

```
@@ -0,0 +1,337 @@
+    hostEl.appendChild(contentEl);
+
+    // Default: no overflow
+    Object.defineProperty(contentEl, 'scrollWidth', { value: 100, configurable: true });
+    Object.defineProperty(contentEl, 'clientWidth', { value: 200, configurable: true });
+  });
+
+  afterEach(() => {
+    jest.useRealTimers();
+    document.body.innerHTML = '';
+    // Remove injected style elements
+    document.querySelectorAll('[data-pds-truncation-tooltip]').forEach(el => el.remove());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)